GameLogic/engine/StandardGame.cs

```diff
@@ -6,180 +6,139 @@
 
 namespace GameLogic
 {
-    public class StandardGame : IStandardGameLogic, IFENSerializable<StandardGameState>
+    public class StandardGame : IStandardGameLogic<StandardGame>
     {
-        /*
-        This class contains a game logic.
-        */
-
-        /// <summary>
-        /// Serialize game object to FEN notation.
-        /// </summary>
-        /// <param name="objectToSerialize">
-        /// The game to serialize.
-        /// </param>        
-        /// <returns>
-        /// The game FEN notation. 
-        /// </returns>        
-        public string SerializeToFEN(StandardGameState objectToSerialize)
+        // This class is the entry point for all commands coming to the GameLogic project.
+        // It is responsible for initial instantiation of the game from FEN notation.
+        // Game state must be valid!!!
+
+        public StandardGameState gameState;
+        public string FENNotation
```

Comment:
I believe it is better to have an extension method called `GetFENNotation()` instead of the `FENNotation` property.

Rationale:
1. StandardGame should not care about its serialization. Therefore there should be no traces of serialization in the class implementation.
2. Method works better than a property here as there are some computations related to the FEN generation. Properties are usually used for very small computations. For example 
```
double DoubledX => 2 * X;
```

Reply:
Do you mean to create an extension class?

```
public static class StandardGameExtension
{
    public static GetFENNotation(this StandardGame game)
    {
        return StandardFENSerializer.SerializeToFEN(gameState);
    }
}
```

Reply:
Yes. Looks good.

Reply:
Fixed!